base/testfiles/tlb0250.tlg

```diff
@@ -39,4 +39,6 @@ Completed box being shipped out [1]
 ....\glue 0.0 plus 1.0fil
 ....\OT1/cmr/m/n/12 1
 ....\glue 0.0 plus 1.0fil
+Non-PDF special ignored!
+<special> header=gnudict.pro
```

Comment:
was there a "test" reason for a specific header file? ie is this still testing something?

Reply:
The test is that specials were lost in the preamble: clearly we are not _loosing_ this one, it's just it does nothingin PDF mode